### ISSUE TITLE
training problem--loss_img_cls

### ISSUE THREAD

**Issue by dingjietao** [ppengtang/pcl.pytorch#61]
I apply the pcl_model to my own dataset, include two classes, and every image have the two classes.
But, when I run the model, I find loss_img_loss_cls falled very fast. When step 1, the loss_img_cls is about 0.7, 
but when step 41, the loss_img_cls fall to 0.05. The loss_img_cls fall too fast. I can not find the reason, 
I would appreciate it if you could help me！！

**Comment by ppengtang** [comment]
Two-class image classification shouldn't be a hard problem and so the loss will be small after a few iterations. Are your detection results reasonable?

**Comment by HMCCMH** [comment]
> I apply the pcl_model to my own dataset, include two classes, and every image have the two classes. But, when I run the model, I find loss_img_loss_cls falled very fast. When step 1, the loss_img_cls is about 0.7, but when step 41, the loss_img_cls fall to 0.05. The loss_img_cls fall too fast. I can not find the reason, I would appreciate it if you could help me！！

Do you have the same problem as followed?
```
[Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 81 / 35000]
                loss: 0.811776, lr: 0.000440 time: 0.578371, eta: 5:36:36
                loss_im_cls: 0.801543, refine_loss0: 0.001459, refine_loss1: 0.002365, refine_loss2: 0.000418
[Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 101 / 35000]
                loss: 0.051120, lr: 0.000467 time: 0.588609, eta: 5:42:22
                loss_im_cls: 0.000066, refine_loss0: 0.024563, refine_loss1: 0.000666, refine_loss2: 0.000001
[Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 121 / 35000]
                loss: 1.797216, lr: 0.000493 time: 0.575597, eta: 5:34:36
                loss_im_cls: 1.725291, refine_loss0: 0.026856, refine_loss1: 0.000002, refine_loss2: 0.000000
[Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 141 / 35000]
                loss: 0.043532, lr: 0.000520 time: 0.574012, eta: 5:33:30
                loss_im_cls: 0.000001, refine_loss0: 0.024328, refine_loss1: 0.000000, refine_loss2: 0.000000
[Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 161 / 35000]
                loss: 0.069948, lr: 0.000547 time: 0.569202, eta: 5:30:30
                loss_im_cls: 0.000001, refine_loss0: 0.026878, refine_loss1: 0.000000, refine_loss2: 0.000000
```

**Comment by dingjietao** [comment]
> 

Sorry for the late reply. I met similar problem, i didn't solve my problem. I changed datasets the problem disappeared.

**Comment by dingjietao** [comment]
> > I apply the pcl_model to my own dataset, include two classes, and every image have the two classes. But, when I run the model, I find loss_img_loss_cls falled very fast. When step 1, the loss_img_cls is about 0.7, but when step 41, the loss_img_cls fall to 0.05. The loss_img_cls fall too fast. I can not find the reason, I would appreciate it if you could help me！！
> 
> Do you have the same problem as followed?
> 
> ```
> [Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 81 / 35000]
>                 loss: 0.811776, lr: 0.000440 time: 0.578371, eta: 5:36:36
>                 loss_im_cls: 0.801543, refine_loss0: 0.001459, refine_loss1: 0.002365, refine_loss2: 0.000418
> [Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 101 / 35000]
>                 loss: 0.051120, lr: 0.000467 time: 0.588609, eta: 5:42:22
>                 loss_im_cls: 0.000066, refine_loss0: 0.024563, refine_loss1: 0.000666, refine_loss2: 0.000001
> [Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 121 / 35000]
>                 loss: 1.797216, lr: 0.000493 time: 0.575597, eta: 5:34:36
>                 loss_im_cls: 1.725291, refine_loss0: 0.026856, refine_loss1: 0.000002, refine_loss2: 0.000000
> [Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 141 / 35000]
>                 loss: 0.043532, lr: 0.000520 time: 0.574012, eta: 5:33:30
>                 loss_im_cls: 0.000001, refine_loss0: 0.024328, refine_loss1: 0.000000, refine_loss2: 0.000000
> [Dec23-21-33-02_ubuntu_step][vgg16_voc2007bolt.yaml][Step 161 / 35000]
>                 loss: 0.069948, lr: 0.000547 time: 0.569202, eta: 5:30:30
>                 loss_im_cls: 0.000001, refine_loss0: 0.026878, refine_loss1: 0.000000, refine_loss2: 0.000000
> ```

Sorry for the late reply. I met similar problem, i didn't solve my problem. I changed datasets the problem disappeared.
